### PR TITLE
feat(frontend): filter repositories that has no issue

### PR DESCRIPTION
**Pull request by up2dul**
# Resolve on issue #56 
Here i filtered the repository list only showed repositories that has issue 

### Before
There is showed rosetta and manusier repo that has no issue
Screenshot 2022-10-09 at 22-18-58 Hacktoberfest Teknologi Umum

### After
Screenshot 2022-10-09 at 22-19-34 Hacktoberfest Teknologi Umum

### CODE REVIEW

File: frontend/src/routes/index.tsx [hunk 1]
```diff
@@ -90,15 +90,17 @@ export default component$(() => {
...
-          <RepositoryCard
-            full_name={mutable(repo.full_name)}
-            html_url={mutable(repo.html_url)}
-            description={mutable(repo.description)}
-            languages={mutable(repo.languages)}
-            issues={mutable(repo.issues)}
-          />
-        ))}
+        {state.filteredRepositories
+          .filter(({ issues }) => issues.length > 0)
```
Comment (elianiva) [inline frontend/src/routes/index.tsx:94]: filternya bukan disini harusnya, tapi disini
https://github.com/teknologi-umum/hacktoberfest/blob/f9eeab86188c75a9ae54ab74229e27c785e97211/frontend/src/services/filter-repositories.ts#L6

**Review by elianiva — CHANGES_REQUESTED** [review]

File: frontend/src/routes/index.tsx (hunk 1 above)
Comment (up2dul) [reply frontend/src/routes/index.tsx:94]: ternyata di file itu udah ada filter-nya ya, tapi gak ngefek apa-apa
image

Comment (elianiva) [reply frontend/src/routes/index.tsx:94]: karena dia nunggu ada filter by labels, harusnya filter itu berlaku meskipun gaada labelsnya

**Comment by aldy505** [comment]
bump, siapa tau lupa

**Comment by aldy505** [comment]
Hello there. I'm gonna close this issue as we'll rewrite the repo soon enough.

**Pull request closed without merging**